Makefile

```diff
@@ -48,18 +29,13 @@ help:
 	@echo "  dev       - prepare a development environment (includes tests)"
 	@echo "  instdev   - prepare a development environment (no tests)"
 	@echo "  install   - install into current Python environment"
-	@echo "  html      - generate docs as standalone HTML files in: $(SPHINXBUILDDIR)/html"
-	@echo "  pdf       - generate docs as PDF (via LaTeX) for paper format: $(SPHINXPAPER) in: $(SPHINXBUILDDIR)/pdf"
-	@echo "  man       - generate docs as manual pages in: $(SPHINXBUILDDIR)/man"
-	@echo "  docchanges   - generate an overview of all changed/added/deprecated items in docs"
-	@echo "  doclinkcheck - check all external links in docs for integrity"
-	@echo "  doccoverage  - run coverage check of the documentation"
```

Comment:
these targets are implicitly still available though.

Reply:
Since we're unlikely to use them I think this is fine?

Reply:
the targets html and doclinkcheck i do use occasionally. i'd keep everything in that might make it easier for contributors to verify their changes.

Reply:
those targets are still available:

```
$ make 
Please use "make <target>" where <target> is one of
  release   - build a release and publish it
  dev       - prepare a development environment (includes tests)
  instdev   - prepare a development environment (no tests)
  install   - install into current Python environment
  clobber   - remove any build products
  build     - build the package
  test      - test from this directory using tox, including test coverage
  publish   - upload to PyPI
  clean     - remove any temporary build products
  dry-run   - perform all action required for a release without actually releasing
Sphinx v3.2.1
Please use `make target' where target is one of
  html        to make standalone HTML files
  dirhtml     to make HTML files named index.html in directories
  singlehtml  to make a single large HTML file
  pickle      to make pickle files
  json        to make JSON files
  htmlhelp    to make HTML files and an HTML help project
  qthelp      to make HTML files and a qthelp project
  devhelp     to make HTML files and a Devhelp project
  epub        to make an epub
  latex       to make LaTeX files, you can set PAPER=a4 or PAPER=letter
  latexpdf    to make LaTeX and PDF files (default pdflatex)
  latexpdfja  to make LaTeX files and run them through platex/dvipdfmx
  text        to make text files
  man         to make manual pages
  texinfo     to make Texinfo files
  info        to make Texinfo files and run them through makeinfo
  gettext     to make PO message catalogs
  changes     to make an overview of all changed/added/deprecated items
  xml         to make Docutils-native XML files
  pseudoxml   to make pseudoxml-XML files for display purposes
  linkcheck   to check all external links for integrity
  doctest     to run all doctests embedded in the documentation (if enabled)
  coverage    to run coverage check of the documentation (if enabled)
```

```
$ make html
Running Sphinx v3.2.1
making output directory... done
loading intersphinx inventory from https://docs.python.org/3.7/objects.inv...
building [mo]: targets for 0 po files that are out of date
building [html]: targets for 1 source files that are out of date
updating environment: [new config] 1 added, 0 changed, 0 removed
reading sources... [100%] index

/home/jon/devel/nir0s/distro/docs/index.rst:23: WARNING: Inline emphasis start-string without end-string.
/home/jon/devel/nir0s/distro/docs/index.rst:57: WARNING: Unknown target name: "uname command output".
looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... done
writing output... [100%] index

generating indices...  genindex py-modindexdone
highlighting module code... [100%] distro

writing additional pages...  searchdone
copying static files... ... done
copying extra files... done
dumping search index in English (code: en)... done
dumping object inventory... done
build succeeded, 2 warnings.

The HTML pages are in docs/_build/html.
```

```
$ make linkcheck
Running Sphinx v3.2.1
making output directory... done
loading pickled environment... done
building [mo]: targets for 0 po files that are out of date
building [linkcheck]: targets for 1 source files that are out of date
updating environment: 0 added, 0 changed, 0 removed
looking for now-outdated files... none found
preparing documents... done
writing output... [100%] index

(line    5) ok        https://docs.python.org/3.7/library/platform.html#platform.dist
(line    5) ok        https://docs.python.org/3.7/library/platform.html#platform.linux_distribution
(line   27) ok        https://github.com/python-distro/distro/tree/master/tests/resources/distros
(line    1) ok        https://docs.python.org/3.7/library/functions.html#bool
(line   10) ok        https://github.com/python-distro/distro
(line   41) redirect  https://github.com/python-distro/distro/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22> - with unknown code to https://github.com/python-distro/distro/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22%3E
(line   31) ok        https://github.com/python-distro/distro/issues
(line    1) ok        https://docs.python.org/3.7/library/stdtypes.html#str
(line    1) ok        https://docs.python.org/3.7/library/stdtypes.html#str
(line    1) ok        https://docs.python.org/3.7/library/stdtypes.html#str
(line    1) ok        https://docs.python.org/3.7/library/stdtypes.html#str
(line  192) ok        https://docs.python.org/3.7/library/shlex.html#module-shlex
(line    5) ok        https://bugs.python.org/issue1322
(line  241) redirect  http://help.ubuntu.com/ - permanently to https://help.ubuntu.com/
(line  279) redirect  https://www.redhat.com/ - permanently to https://www.redhat.com/en
(line  209) redirect  http://www.freedesktop.org/software/systemd/man/os-release.html - with Found to https://www.freedesktop.org/software/systemd/man/os-release.html
(line    8) redirect  http://www.freedesktop.org/software/systemd/man/os-release.html - with Found to https://www.freedesktop.org/software/systemd/man/os-release.html
(line  318) redirect  http://refspecs.linuxfoundation.org/LSB_5.0.0/LSB-Core-generic/LSB-Core-generic/lsbrelease.html - permanently to https://refspecs.linuxfoundation.org/LSB_5.0.0/LSB-Core-generic/LSB-Core-generic/lsbrelease.html
(line  240) redirect  http://www.ubuntu.com/ - permanently to https://ubuntu.com/
(line  280) ok        https://bugzilla.redhat.com/
(line  242) redirect  http://bugs.launchpad.net/ubuntu/ - with Found to https://bugs.launchpad.net/ubuntu/

build succeeded.

Look for any errors in the above output or in docs/_build/linkcheck/output.txt
```

Reply:
FWIW, and perhaps more controversial, I think we should drop all non-sphinx make targets as they mostly duplicate the tox file. Ones that are still useful could move to the tox file.

Reply:
i do contrarily prefer a `Makefile` as primary entrypoint for tasks. and this one can be improved. later.